source/reference/mongotranslate.txt

```diff
@@ -0,0 +1,123 @@
+==================
+``mongotranslate``
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. only:: html
+
+   .. meta::
+      :description: The mongotranslate reference page.
+      :keywords: mongotranslate, mongodb, man page
+
+Description
+-----------
+
+.. only:: (not man)
+
+   .. binary:: mongotranslate
+
+``mongotranslate`` takes a SQL query and returns
+an equivalent MongoDB :manual:`aggregation pipeline </aggregation>`. It
+is designed to help users understand how |bi-short| translates
+SQL queries for use with MongoDB databases.
+
+``mongotranslate`` is a standalone program. It requires
+a :doc:`drdl </reference/mongodrdl>` file generated by
+:binary:`~bin.mongodrdl`, but does not require
+:binary:`~bin.mongosqld` or :binary:`~bin.mongod` to be running at the
+time of execution.
+
+Usage
+-----
+
+``mongotranslate`` has the following syntax:
+
+.. code-block:: javascript
+
+   mongotranslate [--query | --queryFile] <query | queryfile> [options]
+
+Command Line Options
+--------------------
+
+.. program:: mongotranslate
+
+.. include:: /includes/option/option-mongotranslate-query.rst
+
+.. include:: /includes/option/option-mongotranslate-queryFile.rst
+
+.. include:: /includes/option/option-mongotranslate-schema.rst
+
+.. include:: /includes/option/option-mongotranslate-dbName.rst
+
+.. include:: /includes/option/option-mongotranslate-format.rst
+
+.. include:: /includes/option/option-mongotranslate-explain.rst
```

Comment:
Not sure how the order was chosen here but I think `--explain` should go above `--format`, since the `explain` is discussed in the `format` section.

Reply:
Done.